src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.PurchaseHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseHandlerTest {
+    private static OperationHandler purchaseHandler;
+
+    @BeforeClass
+    public static void setUp() {
+        purchaseHandler = new PurchaseHandler();
+        Map<String, OperationHandler> operationHandler = new HashMap<>();
+        operationHandler.put("b", new BalanceHandler());

```

Comment:
remove it from every implementations test classes at all, you don't need it in your logic